README.md

```diff
@@ -42,8 +42,16 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
-
+S -> <empty>
+S -> R
+S -> G
+S -> B
+S -> RSR
+S -> GSG
+S -> BSB
+R -> 'r'
+G -> 'g'
+B -> 'b'
```

Comment:
You're on the right track, but allowing S to be empty breaks it, because the path has to be an odd number of pavers, and these rules allow (for example) "rr"

# -2

also, your can just use r, g, and b as terminals, so

~~~
       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b
~~~